net/gsocket/Makefile

```diff
@@ -0,0 +1,71 @@
+# https://gsocket.io
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gsocket
+PKG_VERSION:=1.4.33
+PKG_RELEASE:=1
+
+#PKG_LICENSE:=BSD-2-Clause
+#PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+# PKG_SOURCE_PROTO:=git
+# PKG_SOURCE_URL:=https://github.com/hackerschoice/gsocket.git
+# PKG_SOURCE_VERSION:=98a85e270c8ee5757134f00636a5358432cd3d83
+
+# Direct file download
+PKG_SOURCE_URL:=https://github.com/hackerschoice/gsocket/releases/download/v$(PKG_VERSION)/
+PKG_HASH:=5bbc850a274b933a4e8b0ac7d5bc8b0527c3eddbaee7f8a9389c284f27a6fe14
+
+# For testing copy to openwrt/dl
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
```

Comment:
If you will take a look at how others Makefiles are done, you will see there is order where this should be.